### PR TITLE
update viselements.json config

### LINKED ISSUE #127: BUG-Dialog syntax not recognized
**Description**
A way to use Dialog is not recognized by Taipy Studio for GUI. It raises a `Missing closing syntax` when in fact dialog can be used like a block.

**How to reproduce**
```
<|{show_dialog}|dialog|
    ...
    jfienf
    ...
|>
```

**Expected behavior**
This markdown shouldn't create an error.

**Runtime environment**
 - Taipy Studio GUI: 0.0.2

### PR DESCRIPTION
**Pull request by dinhlongviolin1**
fix Avaiga/taipy-studio#127, #3

### CODE REVIEW

**Review by FredLL-Avaiga — APPROVED** [review]

File: src/assets/viselements.json [hunk 1]
```diff
@@ -904,6 +904,16 @@
             "type": "str",
             "doc": "Allows the styling of table cells.\nSee <a href=\"#styling\">below</a> for details."
           },
+          {
+            "name": "tooltip",
+            "type": "str",
+            "doc": "The name of the function that must return a tooltip text for a cell.\nSee <a href=\"#cell%20tooltip\">below</a> for details."
```
Comment (FabienLelaquais) [inline src/assets/viselements.json:910]: How does that reflect in Studio?

File: src/assets/viselements.json [hunk 2]
```diff
@@ -904,6 +904,16 @@
...
+          },
+          {
+            "name": "tooltip[<i>col_name</i>]",
+            "type": "str",
+            "doc": "The name of the function that must return a tooltip text for a cell.\nSee <a href=\"#cell%20tooltip\">below</a> for details."
```
Comment (FabienLelaquais) [inline src/assets/viselements.json:915]: Same

**Review by FabienLelaquais — APPROVED** [review]
👌

File: src/assets/viselements.json (hunk 1 above)
Comment (dinhlongviolin1) [reply src/assets/viselements.json:910]: It is reflected as string/markdown string --> will only rendered as a normal string

**Pull request merged**